src/tools/compiletest/src/header/tests.rs

```diff
@@ -553,6 +553,21 @@ fn test_duplicate_revisions() {
     parse_rs(&config, "//@ revisions: rpass1 rpass1");
 }
 
+#[test]
+fn test_forbidden_revisions() {
+    let config: Config = cfg().build();
```

Comment:
Problem [LIMITED 2/2]: so this here should also iter through the 3 test suites that support `FileCheck` annotations to check that it should panic, but that it passes for non-`FileCheck` test suites.